docker/datahub-ingestion/Dockerfile

```diff
@@ -29,7 +29,9 @@ COPY --from=prod-build /datahub-src /datahub-src
 FROM base as prod-install
 COPY --from=prod-codegen /datahub-src/metadata-ingestion /datahub-ingestion
 RUN cd /datahub-ingestion && \
-    pip install ".[all]"
+    pip install ".[all]" && python -m spacy download en_core_web_sm
```

Comment:
they have existing patterns for installing new packages, but I didn't see an example of running commands on packages. I'll take another look though

Reply:
I haven't found a better place for this. But similar to the docker `COPY` command for the privacy mapping CSV, this will be removed when we switch to the python classifier lib